websites/views.py

```diff
@@ -460,7 +461,24 @@ def get_queryset(self):
                 )
             else:
                 queryset = queryset.filter(metadata__resourcetype=resourcetype)
-
+        if published:
+            published = _parse_bool(published)
+            if published is True:
+                queryset = queryset.filter(
+                    website__publish_date__isnull=False,
+                    # if the associated website has been published after
+                    # the content record was created, then it has been published
+                    # at least once
+                    website__publish_date__gt=F("created_on"),
+                )
+            else:
+                # unpublished content is any content where the website has never
+                # been published
+                queryset = queryset.filter(
+                    website__publish_date__isnull=True
+                    # or any content where the website was most recently published
+                    # before it was created
+                ) | queryset.filter(website__publish_date__lt=F("created_on"))
```

Comment:
I think this might give identical results, take or leave:
```python
queryset = queryset.filter(
 Q(website__publish_date__isnull=True) | Q(website__publish_date__lt=F("created_on"))
 )